Makefile

```diff
@@ -61,11 +61,10 @@ ci-e2e:  ## Runs ci e2e tests
 ci-e2e: build check-env
 	$(call ndef, GITHUB_AUTH_TOKEN)
 	mkdir -p bin
-	./scorecard --repo=https://github.com/ossf/scorecard --format json --metadata=openssf > ./bin/results.json
-	ginkgo -p  -v -cover  ./...
 	mkdir -p cache
 	USE_DISK_CACHE=1 DISK_CACHE_PATH="./cache" ./scorecard --repo=https://github.com/ossf/scorecard --show-details --metadata=openssf  --format json > ./bin/results.json
-	ginkgo -p  -v -cover  ./...
+	@sleep 30
```

Comment:
This will be addressed when we fix this issue https://github.com/ossf/scorecard/issues/206